packages/server/src/modules/db/migrate.ts

```diff
@@ -0,0 +1,40 @@
+import * as migrate from 'migrate';
+import { MigrationSet } from 'migrate';
+
+class MockFileStore {
+  save(set: MigrationSet, cb: (err: Error | null) => void) {
+    cb(null);
+  }
+  load(cb: (...args: any[]) => void) {
+    cb(null, { migrations: [] });
+  }
+}
+
+/**
+ * Used to run migrations on test databases.
+ * These migrations are temporary.
+ */
+export function runContainerMigrations() {
+  return new Promise<void>((resolve, reject) => {
+    migrate.load(
+      {
+        migrationsDirectory: './src/application/migrations/migrations',
+        stateStore: new MockFileStore(),
```

Comment:
Why mock file? What does that do?

Reply:
I created this function just to be able to run migrations for each DB in our test container without storing the log file, hence the "mock".